eventuate-local-java-cdc-connector-mysql-binlog/src/main/java/io/eventuate/local/mysql/binlog/MySqlBinaryLogClient.java

```diff
@@ -240,6 +274,7 @@ private void handleWriteRowsEvent(Event event, Optional<BinlogFileOffset> starti
 
     onEventReceived();
     saveOffset(event);
+    publishingException = Optional.empty();
```

Comment:
It think `publishingException = Optional.empty();` should only be done in leadershipStart()

Reply:
ok